### PR TITLE
feat(tutorial): complete step 1

### PR DESCRIPTION
**Pull request by JuanM04**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
   render() {
     return (
-      <div>
-        Hello Carbon! Well, not quite yet. This is the starting point for the
-        Carbon tutorial.
-      </div>
+      <>
+        <TutorialHeader />
+        <Content>
+          <Button>Button</Button>
```
Comment (kellychurchill) [inline src/App.js]: The button component should be replaced with the Switch component: See https://www.carbondesignsystem.com/tutorial/react-step-1#add-routing

File: src/content/LandingPage/LandingPage.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const LandingPage = () => <div>LANDING PAGE</div>;
```
Comment (kellychurchill) [inline src/content/LandingPage/LandingPage.js]: Thie LandingPage const should be replaced with `const LandingPage = () => {
  return <div>LANDING PAGE</div>;
}`

See https://www.carbondesignsystem.com/tutorial/react-step-1#import-and-export-content-pages

File: src/content/RepoPage/RepoPage.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const RepoPage = () => <div>REPO PAGE</div>;
```
Comment (kellychurchill) [inline src/content/RepoPage/RepoPage.js]: This RepoPage const should be replaced with `const RepoPage = () => {
  return <div>REPO PAGE</div>;
};`

See https://www.carbondesignsystem.com/tutorial/react-step-1#import-and-export-content-pages

File: src/index.js [hunk 4]
```diff
@@ -1,10 +1,21 @@
...
 import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+import { HashRouter as Router, Route, Switch } from 'react-router-dom';
+import LandingPage from './content/LandingPage';
+import RepoPage from './content/RepoPage';
+
+ReactDOM.render(
+  <Router>
+    <Switch>
```
Comment (kellychurchill) [inline src/index.js]: The `<Switch>` component should not have replaced the `<App>` component.

File: src/content/LandingPage/LandingPage.js (hunk 2 above)
Comment (JuanM04) [reply src/content/LandingPage/LandingPage.js]: It's the same, but ok

**Comment by kellychurchill** [comment]
👍 Looks good - 🎉 You are one step closer to completion of the Carbon React tutorial and earning the digital credential.

**Pull request closed without merging**